storm/datadog_checks/storm/storm.py

```diff
@@ -160,20 +162,23 @@ def _get_bool(stat_map, default, *components):
     """
     return _g(stat_map, default, _bool, *components)
 
+
 def _get_list(stat_map, *components):
     """ Helper Function to safely get the list value from the map.
 
     :param stat_map: stat map
     :param components: components in order to traverse.
     :return: list of component
     :rtype: list
+    val = _g(s, [], None, 'acked')
```

Comment:
not sure why this is here

Reply:
Accidentally left in from previous iterations :) Incoming commit to remove!